service/worker/workerdeployment/client.go

```diff
@@ -726,11 +752,11 @@ func (d *ClientImpl) updateWithStartWorkerDeploymentVersion(
 			},
 			CreateTime:        now,
 			RoutingUpdateTime: now,
-			CurrentSinceTime:  nil, // not current
-			RampingSinceTime:  nil, // not ramping
-			RampPercentage:    0,   // not ramping
-			DrainageInfo:      nil, // not draining or drained
-			Metadata:          nil, // todo
+			CurrentSinceTime:  nil,                                 // not current
+			RampingSinceTime:  nil,                                 // not ramping
+			RampPercentage:    0,                                   // not ramping
+			DrainageInfo:      &deploymentpb.VersionDrainageInfo{}, // not draining or drained
+			Metadata:          nil,                                 // todo
```

Comment:
I think the client should be able to not mention these values at all and just zero values work fine. otherwise, we are violating the boundaries because client should not have responsibility of keeping workflow local state correct.